Configurations/unix-Makefile.tmpl

```diff
@@ -654,9 +654,11 @@ errors:
 	( cd $(SRCDIR); $(PERL) util/ck_errf.pl -strict */*.c */*/*.c )
 	( cd $(SRCDIR); $(PERL) util/mkerr.pl -internal )
 	( cd $(SRCDIR)/engines; \
-	  for e in *.ec; do \
-	      $(PERL) ../util/mkerr.pl -conf $$e -rebuild *.c; \
-	  done )
+          $(PERL) ../util/mkerr.pl -conf e_capi.ec e_capi.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_dasync.ec e_dasync.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_ossltest.ec e_ossltest.c ; \
+          cd afalg; \
+          $(PERL) ../../util/mkerr.pl -conf e_afalg.ec e_afalg.c )
```

Comment:
If anything, the GOST project taught us that we can't rely the the `.ec` file to tell us exactly which C files are involved.  That's why it was done this way (actually for a long time, I think we foresaw the possibility long before the GOST engine came along).  Like I said, though, it's perfectly possible to change things so we declare the relationship more explicitely... that would have to be with another variable in the `build.info` files.  I'm pondering it...